ci/deploy/npm.py

```diff
@@ -0,0 +1,164 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+import argparse
+import json
+import logging
+import os
+import shutil
+import urllib.parse
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Optional
+
+import requests
+
+from materialize import spawn
+
+logging.basicConfig(
+    format="%(asctime)s [%(levelname)s] %(message)s",
+    level=os.environ.get("MZ_DEV_LOG", "INFO").upper(),
+)
+logger = logging.getLogger(__name__)
+
+PROJECT_PATHS = ["src/sql-lexer"]
+ROOT_PATH = Path(".")
```

Comment:
```suggestion
from materialize import ROOT
```

(But you'll want to import that above.)